src/main/java/uk/org/tombolo/field/aggregation/MapToContainingSubjectField.java

```diff
@@ -65,7 +65,7 @@ private Subject getSubjectContainingSubject(Subject subject) throws Incomputable
                     "Subject %s is contained by %d subjects of type %s (%s), but should be contained by 1 only",
                     subject.getName(),
                     subjectsContainingSubject.size(),
-                    containerSubjectType.getLabel(),
+                    containerSubjectType == null ? "UNKNOWN" : containerSubjectType.getLabel(),
                     subjectsContainingSubject.stream().map(Subject::getName).collect(Collectors.joining(", "))));
```

Comment:
Could see couple potential error/exceptions with this block of code.
1) What is the size of "subjectsContainingSubject" is greater than 1. I think we could change it to "!subjectsContainingSubject.isEmpty()"
2) If "containerSubjectType" can be null, then there is no point making the query to SubjectUtils because that would return empty list or null. I think null check on "containerSubjectType" should happen before making the query to SubjectUtils.

Reply:
> What is the size of "subjectsContainingSubject" is greater than 1. I think we could change it to "!subjectsContainingSubject.isEmpty()"

I think the idea is that this size check tries to see if the our subject is contained in more than one parent or no parent at all. In both cases we need to launch an IncomputableFieldException. So checking that is empty or not is not enough, I believe. We could separate this 2 cases though and launch a different message to help the user navigate better in the 2 cases.

> If "containerSubjectType" can be null, then there is no point making the query to SubjectUtils because that would return empty list or null. I think null check on "containerSubjectType" should happen before making the query to SubjectUtils.

Yeah, this makes sense I will check it before.

Reply:
Yup my bad i didn't read the exception message :-(